### PR TITLE
Complete nightly test for multi vlan

### PR DESCRIPTION
**Pull request by mdharamadas1**
Fix for https://github.com/vmware/vic-ui/issues/406

### CODE REVIEW

File: tests/manual-test-cases/Group5-Interoperability-Tests/5-12-Multiple-VLAN.robot [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+# limitations under the License
+
+*** Settings ***
+Documentation  Test 5-12 - Multiple VLAN
+Resource  ../../resources/Util.robot
+Suite Setup  Wait Until Keyword Succeeds  10x  10m  Multiple VLAN Setup And Install OVA With UI Plugin
+Suite Teardown  Run Keyword And Ignore Error  Nimbus Cleanup  ${list}
+
+*** Keywords ***
+Multiple VLAN Setup And Install OVA With UI Plugin
```
Comment (mhagen-vmware) [inline tests/manual-test-cases/Group5-Interoperability-Tests/5-12-Multiple-VLAN.robot]: this does not install OVA nor UI plugin

Comment (mhagen-vmware) [reply tests/manual-test-cases/Group5-Interoperability-Tests/5-12-Multiple-VLAN.robot]: nor should it

File: tests/manual-test-cases/Group5-Interoperability-Tests/5-12-Multiple-VLAN.robot [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    Click Next Button
+    # ops-user
+    Input Ops User Name  %{TEST_USERNAME}
+    Input Ops User Password  %{TEST_PASSWORD}
+    Click Next Button
+    # summary
+    Click Finish Button
+    Unselect Frame
+    Wait Until Page Does Not Contain  VCH name
+    # retrive docker parameters from UI
```
Comment (mhagen-vmware) [inline tests/manual-test-cases/Group5-Interoperability-Tests/5-12-Multiple-VLAN.robot]: retrieve

File: tests/manual-test-cases/Group5-Interoperability-Tests/5-12-Multiple-VLAN.robot (hunk 1 above)
Comment (mdharamadas1) [reply tests/manual-test-cases/Group5-Interoperability-Tests/5-12-Multiple-VLAN.robot]: Corrected keyword naming

**Pull request merged**